Pipelines/oat-pr.yml

```diff
@@ -0,0 +1,72 @@
+# Azure Pipelines
+# https://aka.ms/yaml
+
+name: OAT_PR_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+trigger: none
+pr:
+  branches:
+    include:
+    - main
+
+resources:
+  repositories:
+    - repository: templates
+      type: git
+      name: SecurityEngineering/OSS-Tools-Pipeline-Templates
+      ref: refs/tags/v1.0.6
+
+stages:
+- stage: Test
+  dependsOn: []
+  jobs:
+  - template: dotnet-test-job.yml@templates
+    parameters:
+      jobName: 'dotnet_test_windows'
+      dotnetVersion: 7.0.x
+      dotnetVersionLTS: 6.0.x
+      vmImage: 'oss-tools-win2022_1es-managed'
+      projectPath: 'OAT.Tests/OAT.Tests.csproj'
+  - template: dotnet-test-job.yml@templates
+    parameters:
+      jobName: 'dotnet_test_ubuntu'
+      dotnetVersion: 7.0.x
+      dotnetVersionLTS: 6.0.x
+      vmImage: 'ubuntu-22.04-image-base'
+      projectPath: 'OAT.Tests/OAT.Tests.csproj'
+
+- stage: SDL
+  dependsOn: []
+  jobs:
+  - template: sdl-job.yml@templates
+
+- stage: Build
+  dependsOn:
+  - Test
+  jobs:
+  - template: nuget-build-job.yml@templates
+    parameters:
+      jobName: 'pack_lib'
+      buildConfiguration: 'Release'
+      dotnetVersion: '7.0.x'
```

Comment:
Nit: its nice to have these version numbers specified as parameters at the top of the script once like in the previous version for when we change the target frameworks for the package.

Reply:
Done. Making sure everything works in PR pipeline then will add to Release as well.